internal/mocks/info.go

```diff
@@ -0,0 +1,356 @@
+// Code generated by mockery v2.14.0. DO NOT EDIT.
```

Comment:
Don't you need to add this to scripts/regenerate_mocks.sh?

Reply:
Didn't find the PChain mock there either. In fact, it's not straight forward as the other ones because both are named just Client and needs a bit of editing. So concluded that's the reason PClient is not there.

I think it can be added with different flags than the other mocks.

Reply:
Hmm actually the reason is that these reside in a different repository. `mockery` seems not able to handle that very well;
`outside main module or its selected dependencies`.

Looks like mocks for other repo objects need to be created manually. That seems the reason why the platformvm Client was also not in this file.

Reply:
It might be nice if we could add this to the readme somehow/somewhere. Might be confusing for us later if the avalanchego side changes.

Reply:
The mocks are in an `internal` package - I think it'd be weird to add this to the public README. 

I'll add a new README to the `internal/mocks` package and also add a note to the regeneration script.